sections/semantics-textlevel.include

```diff
@@ -2946,20 +2946,22 @@
   </div>
 
   <div class="example">
-    If the <{b}> element is used to draw attention to the content, the <{bdi}> element is not needed.
+    The <{bdi}> element is a shorthand for <{span/dir}> with a value of <code>auto</code>, so the <code>dir</code> attribute can be applied to another element to achieve the same result.
+
+    For example, the <{b}> element may be used to draw attention to part of the content:
 
     <figure>
       <img src="images/sample-not-bdi.png" />
       <figcaption>If the <{bdi}> element were to be replaced by a <{b}> element, the username would confuse the bidirectional algorithm and the third bullet would end up saying "User 3 :", followed by the Arabic name (right-to-left), followed by "posts" and a period.</figcaption>
     </figure>
 
-    The <{bdi}> element is a shorthand for <{span/dir}> with a value of <code>auto</code>, so the <code>dir</code> attribute can be applied to the <code>b</code> element instead.
+    In this case, the <{bdi}> and <{b}> elements could both be used, but a shorter way to achieve the same result is to apply the <code>dir</code> attribute directly to the <b> element:
```

Comment:
Here I would say: "In this case, where there is an existing element available, the shortest way to achieve the same result is to ..."

What that does is avoid the suggestion that `<b><bdi>` is sensible, because it really isn't, and yet many people think they have to do that. 

You might add a sentence at the end of the example, however, to say something like: "If there isn't an existing element in place, then the shortest way is to use just `<bdi>` around the text."